resources/osl_ssh_key.rb

```diff
@@ -0,0 +1,40 @@
+provides :osl_ssh_key
+resource_name :osl_ssh_key
+unified_mode true
+
+default_action :add
+
+property :key_name, String, name_property: true
+property :key, String, required: [:add]
```

Comment:
`:content` might be a better name for this property.

Reply:
okay